### PR TITLE
fix issue [5089](https://github.com/ant-design/ant-design/issues/5089)

### PR DESCRIPTION
**Pull request by xieguanglei**
fix issue [5089](https://github.com/ant-design/ant-design/issues/5089)

### CODE REVIEW

File: components/tabs/index.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export interface TabsProps {
   style?: React.CSSProperties;
   prefixCls?: string;
   className?: string;
-  animated?: boolean;
+  animated?: any;
```
Comment (afc163) [inline components/tabs/index.tsx]: any 不是很合适。

Comment (xieguanglei) [inline components/tabs/index.tsx]: agree, i'll update

Comment (xieguanglei) [inline components/tabs/index.tsx]: updated

File: components/tabs/index.tsx [hunk 2]
```diff
@@ -97,6 +100,13 @@ export default class Tabs extends React.Component<TabsProps, any> {
       onTabClick,
       animated,
     } = this.props;
+
+    let {inkBarAnimated, tabPaneAnimated} = typeof animated === 'object' ? {
+        inkBarAnimated: !!animated.inkBar, tabPaneAnimated: !!animated.tabPane,
+      } : {
+        inkBarAnimated: !!animated, tabPaneAnimated: !!animated,
+      };
```
Comment (afc163) [inline components/tabs/index.tsx]: 1. wrong code indent.
2. `!!` seems to be unnecessary.

**Comment by afc163** [comment]
Oh, you should PR to feature-2.9 cause it is a new feature.

**Comment by xieguanglei** [comment]
not using rebase and mess my branch up ... I'll pull a new request

**Pull request closed without merging**